lib/Components/SelectHardware.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Header, Button } from '@bpanel/bpanel-ui';
+import styles from '../styles';
+
+import { safeEval, preventDefault } from '../utilities';
+import { PLUGIN_NAMESPACE } from '../constants';
+
+class SelectHardware extends Component {
+  constructor() {
+    super();
+  }
+
+  static get propTypes() {
+    return {}
+  }
+
+  render() {
+
+    const {
+      handleSelect,
+      hardwareAccountInput,
+      selectedXPUB
+    } = this.props;
+
+    return (
+      <div>
+        <div className="row">
+          <div className="col-sm">
+            <Header type="h3">Insert Hardware Wallet</Header>
+          </div>
+          <div className="col-sm">
+            <input
```

Comment:
I didn't think that component supported `type="number"`

Reply:
Since this is really just a styled html input, it should still work: https://github.com/bpanel-org/bpanel-ui/blob/master/src/components/Input/Input.js

But this component really isn't very robust right now imo, so I'm happy either way.